src/engine/source/cmds/src/cmdRun.cpp

```diff
@@ -0,0 +1,163 @@
+#include "cmds/cmdRun.hpp"
```

Comment:
Why does the CMD module include the other modules and initialize them, I think it would be necessary to add this module to the design document and rename it. cmd is not just a parser, it is a new module in charge of initializing and configuring the  Engine.

Reply:
I don't understand, as you said cmds are not parsers, are commands. When the API is implemented this structure will probably change. They only build the Engine or the required functionality of it to perform the different tasks (run engine as in production, run engine in test mode and generate the graphs).